Library/Homebrew/cask/lib/hbc/download_strategy.rb

```diff
@@ -64,11 +64,6 @@ def clear_cache
   end
 
   class CurlDownloadStrategy < AbstractDownloadStrategy
```

Comment:
It would be good to try and unify this code with Homebrew's considering it was a fork in the past.